### DISCUSSION TITLE
Aplicación de Tracking GPS para móviles

### DISCUSSION THREAD

**Discussion by eusonlito** [eusonlito/GPS-Tracker#22] [General]
Hola @kikearcos9999, muevo el mensaje para un issue ya que es más sencillo de realizar seguimiento.

> Saludos;
> 
> Gracias por tu respuesta
> 
> Estoy instalando tu aplicacion en digitalocean, veo muy interesante tu aporte, encontre en amazon este dispositivo lo estoy trayendo para hacer pruebas:
> 
> https://www.amazon.com/SinoTrack-4G-ST-901L-Waterproof-Motorcycle/dp/B09QYG17S8/ref=psdc_617650011_t2_B0C4P5PW6C
> 
> De lo que me comentaste en el anterior correo, OSMAND es una app mobil, segun entiendo con ella puedo hacer seguimiento en linea sin necesidad de un dispositivos gps, estoy en lo correcto.......utilizaria solo mi telefono movil...verdad....?
> 
> (Si deseas integrar la conexión con un móvil, por ejemplo OsmAnd es el más sencillo, ya que es un JSON con la latitud, longitud, tiempo y velocidad.)
> 
> 
> Los equipos que te detallo a continuación los tengo en mi poder uno de cada uno, esperaba hacer algo con ellos, estos equipos entiendo que estan homologados para mi pais. por que aqui los utilice ya con (https://github.com/traccar/traccar) , pero mi intencion es hacer algo personalizado y tu proyecto tiene lo que necesito, de ser factible un manual donde pueda yo integrar la trama de estos dispositivo, claro todo trabajo implicaría un pago y con gusto pagaría
> 
> equipos que en este momento los tengo
> 
> https://www.amazon.com/Magnetic-Vehicle-Tracker-Tracking-Positioning/dp/B0BHJ21ZB3?pd_rd_w=WXw5d&content-id=amzn1.sym.c388ca75-4c14-4d9d-947d-5dcde63263f5&pf_rd_p=c388ca75-4c14-4d9d-947d-5dcde63263f5&pf_rd_r=PTGX4CDQZR93AWNH2JS1&pd_rd_wg=q5Tr2&pd_rd_r=eb9b8640-bae5-40e4-9745-83fc9bd17355&pd_rd_i=B0BHJ21ZB3&psc=1&ref_=pd_bap_d_csi_prsubs_2_i
> 
> https://www.amazon.com/BAANOOL-2G-Intelligent-Subscription-BAANOOL-303F/dp/B0C4P5PW6C/ref=sr_1_1?crid=2CVS0UG3OCVL7&keywords=coban%2Bcar%2Bgps%2Btracker%2B303f&qid=1695826452&sprefix=coban%2Bgps%2B303%2Caps%2C160&sr=8-1&th=1
> 
> Trabajo para una entidad publica aca en mi pais Municipio pequeño en la provincia de cotopaxi- ecuador el canton se llama San Miguel de Salcedo, actualmente me dieron cambio administrativo al terminal terrestre llevo aca un mes, el proyecto que quiero presentar es en el area de TRANSPORTE PUBLICO, quiero enlazar tu proyecto hasta una app mobil personalizada donde estudiantes pueden rastrear el tranporte publico y asi poder estar atento a las paradas. eso basicamente quiero hacer un prototipo a ver como va esto. se algo de laravel, vengo de php puro no al nivel que tu tienes, tengo un nivel basico de laravel, el proyecto es el ambito de mi trabajo.

> Saludos;
> 
> OsmAnd es una app para telefonos móbiles, funciona como un dispositivo gps en linea, si entiendo bien puedo utilizar OsmAnd para simular un dispositivo GPS, estoy bien o entendi mal.

**Comment by eusonlito** [comment]
La aplicación que buscas es GPSLogger: https://github.com/mendhak/gpslogger que te permite configurar un servidor remoto donde realizar el tracking de un dispositivo de forma muy sencilla.

Puedes configuarla para ser compatible con el protocolo OsmAnd soportado por el servidor mediante la siguiente configuración de URL Personalizada:

```
http://DOMINIO:PUERTO/?lat=%LAT&lon=%LON&timestamp=%TIMESTAMP&speed=%SPD&direction=%DIR&id=%SER&maker=gpslogger&hdop=1
```

De este modo podrás registrar las localizaciones mediante un teléfono móvil.

Si deseas un dispositivo barato y que funciona bastante bien, el Sinotrack ST-906L está soportado, el resto son genéricos que no tienen soporte ni siquiera en traccar.

Saludos!

**Reply by kikearcos9999** [reply]
saludos;

La app GPS Tracker, esta corriendo y funcionando actualmente con el protocolo osmand y la url personalizada la cual me sugeriste , aun que no le encuentro aun la configuración ideal en la app gpslogger, pero esta funcionando (con el protocolo osmand hay periodos de 30 minutos que no envia informacion luego si guarda información, no se si sera cuestion de la configuración del gpslogger ).

GPS Tracker platform for Sinotrack ST-90x, OsmAnd and Queclink devices, donde puedo encontrar mas información, sobre la app GPS Tracker,  sobre todo que equipos puede soportar en los protocolos actuales, me sugeriste Sinotrack ST-906L lo estoy trayendo para ecuador en mi pais no lo encontre, que otros dispositivos y en que protocolos puedo  instalar

**Reply by eusonlito** [reply]
Es muy probable que el GPSLogger se quede sin cobertura en algunos lugares, habría que revisar los logs de la plataforma si está recibiendo registros. También puede ser la configuración de ahorro de energía del teléfono, que pausa la aplicación cuando pasa a segundo plano.

En cuanto a dispositivos, es complicado encontrar distribuidores locales de Sinotrack o Queclink, pero a través de Aliexpress o Amazon seguro que puedes conseguirlos sin problema.

**Reply by eusonlito** [reply]
Otra posibilidad es que estés enviando localizaciones con el dispositivo parado. En ese caso la plataforma no añade registros cuando la distancia con la posición anterior sea menos de 10 metros. Puedes cambiar esto en `Configuración` > `position_filter_distance` con el valor que prefieras (o `0` si no quieres descartar ninguna posición incluso parado). Si lo desactivas es probable que te genere un único viaje con todas las posiciones.
